docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -174,40 +212,66 @@ Any configuration setting from a service's `configuration.yaml` file can be over
+
+Service configuration overrides are applied at two levels, which are **common** and **private**.
+
+##### Common Configuration Overrides
+
+When using the Configuration Provider, the common configuration overrides are applied to the **core-common-config-bootstrapper** service. This service is solely responsible with seeding the Configuration Provider with the common configuration. These common configuration overrides are only applied when the **core-common-config-bootstrapper** service reads the common configuration from file and pushes the values in to the Configuration Provider. The  **core-common-config-bootstrapper** service only does this if the Configuration Provider doesn't already have the common configuration. 
+
+When not using the Configuration Provider, the common configuration is provided from the file specified by [`-cc/--commonConfig` command line option](../CommonCommandLineOptions/#common-config). The common configuration overrides are applied when this file is loaded. This occurs on every time a service starts. These common overrides must be set on the individual services rather than a central service since each service is loading the common configuration file. 
 
-    CONFIG : Clients:
-               core-data:
-                 Host: "localhost"
-    ENVVAR : CLIENTS_CORE_DATA_HOST=edgex-core-data    
-    ```    
+##### Private Configuration Overrides
 
-### SecretStore Overrides
+Each service has private configuration which is initially loaded from file. Private configuration overrides are only applied when the private configuration is load from file. These private configuration overrides must be set on the individual services since each service is loading its own private configuration file. 
+
+When using the Configuration Provider,  the values are pushed to the service's private section in the Configuration Provider. Once the private file and overrides have been used to seed the Configuration Provider, they are no longer used. The services will pull their private configuration directly from the Configuration Provider on future start-ups.
+
+When not using the Configuration Provider, the private overrides are applied when the configuration is load from this file and then used by the service. This occurs on every time a service starts.
+
+!!! important
+    Private configuration overrides are only applied to configuration settings that exist in the service's private configuration file.
+
+### SecretStore Configuration Overrides
 
 !!! edgey "EdgeX 3.0"
-    For EdgeX 3.0 the **SecretStore** configuration has been removed from each service's configuration files. It now has default values which can be overridden with environment variables.
+    For EdgeX 3.0 the **SecretStore** configuration has been removed from each service's configuration files and the Configuration Provider. It now has default values which can be overridden with [Configuration Overrides](#configuration-overrides).
```

Comment:
This was already covered in CommonConfiguration.md line 36:
> For EdgeX 3.0 the **SecretStore** configuration has been removed from each service's configuration files. It now has default values which can be overridden with environment variables. See the [SecretStore Overrides](../CommonEnvironmentVariables/#secretstore-configuration-overrides) section for more details.

Reply:
Ok, removed